components/brave_wallet_ui/components/shared/add-custom-token-form/add-custom-token-form.tsx

```diff
@@ -107,12 +108,29 @@ export const AddCustomTokenForm = (props: Props) => {
   } = useTokenInfo(
     getBlockchainTokenInfo,
     combinedTokensList,
-    customAssetsNetwork || selectedNetwork
+    customAssetsNetwork
   )
   const {
     onAddCustomAsset
   } = useAssetManagement()
 
+  const { data: matchedCoingeckoId } = useGetCoingeckoIdQuery(
+    customAssetsNetwork && tokenContractAddress
```

Comment:
should we add a min-length check for the address to prevent looking this up before the entire address is typed-in?

Reply:
Good idea. I can do a stricter check with regex.